deepchem/metrics/__init__.py

```diff
@@ -181,7 +182,12 @@ def compute_metric(self, y_true, y_pred, w=None, n_classes=2, filter_nans=True):
       if filter_nans:
         computed_metrics = np.array(computed_metrics)
         computed_metrics = computed_metrics[~np.isnan(computed_metrics)]
-      return self.task_averager(computed_metrics)
+      if self.compute_force_metrics:
```

Comment:
This should be something like "compute_energy_metric" since we are printing out the force error but only passing back the energy error metric.  This is used where we fit the energy and gradient with multitask models but averaging the metrics of force and energy didn't make much sense